apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -2223,13 +2235,29 @@ input[type="checkbox"]:after {
     background: #fff;
     cursor: pointer;
     border-radius: 2px;
+    visibility: visible;
 }
 
 input[type="checkbox"]:checked:after {
     background-color: $border-blue;
     border-color: $border-blue;
 }
 
+
+
+
+
+//   input[type="checkbox"]::before {
```

Comment:
Avoid introduction of commented code and empty lines

Reply:
Removed empty lines